include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -93,6 +93,14 @@ struct __accessor_traits_impl<sycl::accessor<_T, _Dim, _AccMode, _AccTarget, _Pl
     using value_type = typename sycl::accessor<_T, _Dim, _AccMode, _AccTarget, _Placeholder>::value_type;
 };
 
+#if _ONEDPL_LIBSYCL_VERSION >= 60000
+template <typename _T, int _Dim>
+struct __accessor_traits_impl<sycl::local_accessor<_T, _Dim>>
```

Comment:
I would remove `__accessor_traits` and `__accessor_traits_impl` at all.
Because it used just in one place and there type definition may be re-written via `auto`.

Reply:
I think your proposal does not quite correspond to the goal of this PR.
May be we do this in separate PR ?

Reply:
It makes sense removing `__accessor_traits `and `__accessor_traits_impl` here because I tend to the proposed changes are not quite correct...And doing "correct" ones is meaningless, because just removing unnecessary code/classes is better here, I believe.

Reply:
Fixed